Windows build defaults to daemonising & doesn't provide -D switch

```
C:\remote_syslog>remote_syslog.exe -h
Usage of remote_syslog.exe:
  -c, --configfile="/etc/log_files.yml": Path to config
      --debug-log-cfg="": the debug log file
  -d, --dest-host="": Destination syslog hostname or IP
  -p, --dest-port=0: Destination syslog port
      --eventmachine-tail=false: No action, provided for backwards compatibility

  -f, --facility="user": Facility
      --hostname="": Local hostname to send from
      --log="<root>=INFO": "logging configuration <root>=INFO;first=TRACE"
      --new-file-check-interval={0}: How often to check for new files
      --no-eventmachine-tail=false: No action, provided for backwards compatibility
      --pid-file="": Location of the PID file
  -s, --severity="notice": Severity
      --tcp=false: Connect via TCP (no TLS)
      --tls=false: Connect via TCP with TLS
```

and

```
C:\remote_syslog>remote_syslog.exe -D
unknown shorthand flag: 'D' in -D

C:\remote_syslog>remote_syslog.exe --no-detach=true
unknown flag: --no-detach
```

Running it normally:

```
C:\remote_syslog>remote_syslog.exe
panic: cannot daemonize on windows
```

--- a/config_manager.go
+++ b/config_manager.go
@@ -152,6 +152,8 @@ func (cm *ConfigManager) parseFlags() {
 	pflag.IntVarP(&cm.Flags.DestPort, "dest-port", "p", 0, "Destination syslog port")
 	if utils.CanDaemonize {
 		pflag.BoolVarP(&cm.Flags.NoDaemonize, "no-detach", "D", false, "Don't daemonize and detach from the terminal")
+	} else {
+		cm.Flags.NoDaemonize = true
 	}
 	pflag.StringVarP(&cm.Flags.Facility, "facility", "f", "user", "Facility")
 	pflag.StringVar(&cm.Flags.Hostname, "hostname", "", "Local hostname to send from")

